docs/frontend/frontend-development.md

```diff
@@ -1,20 +1,117 @@
 # Frontend development
 
-The base scss file is placed in the sass folder of the FrameworkCoreBundle. The
-layout is based on Bootstrap, so don't create new components when there's
+## Sass
+
+The styling of the FrameworkCoreBundle is base in a separated npm package, namely [FrameworkStylepackage](https://github.com/sumocoders/frameworkStylePackage).
+
+The base scss file is `style.scss` and is placed in `/assets/styles` in the FrameworkCoreBundle.
+There is also `style-dark.scss` for the dark theme. The last file that is by default in this folder is `mail.scss`.
```

Comment:
The last file that is by default in this folder is → The last default file in this folder is

Reply:
fixed in ce0d873